src/components/organisms/Header/DesktopHeaderMenu.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { Group, HoverCard } from '@mantine/core';
+import { clsx } from 'clsx';
+
+import { ILink } from '@/interfaces/ILink';
+import { Link } from '@/navigation';
+
+import styles from './Header.module.scss';
+
+interface Props {
+  activeLink: string;
+  links: ILink[];
+  onClickLink: (url: string) => void;
+}
+
+const DesktopHeaderMenu: React.FC<Props> = (props) => {
```

Comment:
I can see that we have already declared some components out of this PR scope with `React.FC<Props>` (e.g. `ExhibitNotFoundMessage`). Do we want to keep using approach with `props: Props` without component explicit type declaration?
In this case, I am afraid, it might be easily to face with TS compiler issues due to strict mode and automatic type of component definition